packages/config/package.json

```diff
@@ -4,9 +4,16 @@
   "main": "index.js",
   "license": "MIT",
   "files": [
-    "index.js"
+    "eslint-preset.js",
```

Comment:
Isn't this on its own package now (`eslint-config-commons-ui`)?

Reply:
@kilemensi Yea, um, just removed them